neard/src/config.rs

```diff
@@ -694,6 +694,17 @@ fn state_records_account_with_key(
     ]
 }
 
+/// Generate validator key if account is not `None`.
```

Comment:
This condition looks odd to me. I suggest the method takes AccountId and Option is handled outside of the method.

BTW, having an empty-string AccountId looks error-prone to me. Can we have it resolved to None immediately on receiving an empty string instead of having it in an invalid state?